components/docs-chef-io/content/automate/ha_backup_restore_object_storage.md

```diff
@@ -40,7 +47,18 @@ This section provides the pre-backup configuration required to back up the data
 - `chown -RL hab:hab /hab/svc/automate-ha-opensearch/config/opensearch.keystore` (Setting hab:hab permission)
 - `curl -k -X POST "https://127.0.0.1:9200/_nodes/reload_secure_settings?pretty" -u admin:admin` (Command to load the above setting)
 
-The final output after running the curl command on all nodes is given below:
+#### For _GCS_
+This section provides the pre-backup configuration required to back up the data on GCP Object Storage. The steps to set a secret key using commands are given below:
+
+1. Log in to **all** the OpenSearch nodes and follow the steps on all the OpenSearch nodes.
+
+- `export OPENSEARCH_PATH_CONF="/hab/svc/automate-ha-opensearch/config"`
+- `export GCS_SERVICE_ACCOUNT_FILE_PATH="/path/to/googleServiceAccount.json"` (Provide the file path of your googleServiceAccount.json file)
```

Comment:
Just add a dummy json, how it looks like 
in the docs

Reply:
Done.
Added a new section to show the template and added a hyperlink for the same in the 2nd point of this section [configuring backup for GCS](https://deploy-preview-8213--chef-automate.netlify.app/automate/ha_backup_restore_object_storage/#for-_gcs_)